.github/workflows/dockerimage.yml

```diff
@@ -0,0 +1,40 @@
+
+name: "Build and push release docker image"
+
+on:
+  push:
+    tags:
+      - "v*"
+
+permissions:
+  contents: read
+  packages: write
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      
+      - name: Log in to GitHub container registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN  }}
+      
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@98669ae865ea3cffbcbaa878cf57c20bbf1c6c38
+        with:
+          images: ghcr.io/${{ github.repository }}
```

Comment:
Souldn't this be called "labels"? That's how the output is referenced below.

Reply:
It's not only labels but also tags

Reply:
Oh, nevermind. I got confused by the names of the inputs for this action, and its outputs down below.